examples/region_sharding/101_initial_cluster.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Copyright 2019 The Vitess Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# this script brings up zookeeper and all the vitess components
+# required for a single shard deployment.
+
+source ./env.sh
+
+# start topo server
+if [ "${TOPO}" = "zk2" ]; then
+	CELL=zone1 ./scripts/zk-up.sh
+elif [ "${TOPO}" = "k8s" ]; then
+	CELL=zone1 ./scripts/k3s-up.sh
+else
+	CELL=zone1 ./scripts/etcd-up.sh
+fi
+
+# start vtctld
+CELL=zone1 ./scripts/vtctld-up.sh
+
+# start vttablets for main keyspace. we start only one tablet each (master)
+CELL=zone1 TABLET_UID=100 ./scripts/mysqlctl-up.sh
```

Comment:
Q: we are keeping all shards in the same cell. Will it make the example better to create multiple cells and put each shard in a different cell?

Reply:
Yeah. I think it will be important to showcase multi-cell.